doc/source/WorkingWithThePythonScriptingAPI/TutorialQueryingAScene/index.md

```diff
@@ -209,16 +209,16 @@ print g["option:render:camera"].value
 print g["option:render:resolution"].value
 ```
 
-There are a couple of things to note here. Firstly, although the `out` plug appears as a single plug in the GraphEditor, it actually has several child plugs, which allow different aspects of the scene to be queried. We accessed the `globals` plug using dictionary syntax, and then retrieved its value using the getValue() method. The result was an IECore::CompoundObject which we can pretty much treat like a dictionary, with the minor annoyance that we need to use `.value` to actually retrieve the final value we want.
+There are a couple of things to note here. Firstly, although the Out plug appears as a single plug in the Graph Editor, it actually has several child plugs, which allow different aspects of the scene to be queried. We accessed the `globals` plug using dictionary syntax, and then retrieved its value using the getValue() method. The result was an IECore::CompoundObject which we can pretty much treat like a dictionary, with the minor annoyance that we need to use `.value` to actually retrieve the final value we want.
 
-The `"option:render:camera"` globals entry tells us that the user wants to render through a camera called `"/world/camera"`, so let's try to retrieve the object representing the camera. Just as the globals within the scene were represented by a `globals` plug, the objects are represented by an `object` plug. Maybe we can get the camera out using a simple getValue() call as before?
+The `option:render:camera` globals entry tells us that the user wants to render through a camera called `/world/camera`, so let's try to retrieve the object representing the camera. Just as the globals within the scene were represented by a `globals` plug, the objects are represented by an `object` plug. Maybe we can get the camera out using a simple getValue() call as before?
```

Comment:
`object` plug backticks.

Reply:
In the context of an API doc, it's easiest format code as code. They're interfacing with a dictionary in the Python Editor, not any of the other editors.

Reply:
Cool, so I guess `getValue()` needs them too?

Reply:
Oops, there were a few more, too.